ping/datadog_checks/ping/ping.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under Simplified BSD License (see LICENSE)
+
+import platform
+import re
+
+from datadog_checks.checks import AgentCheck
+from datadog_checks.errors import CheckException
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+
+
+class PingCheck(AgentCheck):
+
+    SERVICE_CHECK_NAME = 'network.ping.can_connect'
+
+    def _load_conf(self, instance):
+        # Fetches the conf
+        timeout = float(instance.get('timeout', 4))
```

Comment:
The timeout parameter is in second and the ping command expect milliseconds, we need to convert it to ms somewhere, here could be a good place for that

Reply:
🤔 I'm looking at the man ping command on a CentOS box and I'm seeing:
```
-W timeout
              Time to wait for a response, in seconds.
```
https://linux.die.net/man/8/ping which also seems to mention seconds. However, I am getting conflicting info on other sites...

Reply:
You're right I'm also getting confusing information, let's make sure we have something compatible with everything then.

Reply:
Looks like Windows uses milliseconds whereas Linux use seconds. 
Windows: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/ping
Linux: man link above. 
I can update the check to reflect that indeed.